src/main/java/com/uid2/admin/job/model/JobInfo.java

```diff
@@ -0,0 +1,29 @@
+package com.uid2.admin.job.model;
+
+import java.time.Instant;
+
+public class JobInfo {
+
+    private final String id;
+    private final boolean executing;
+    private final Instant executionTime;
+
+    public JobInfo(String id, boolean executing) {
+        this.id = id;
+        this.executing = executing;
+        this.executionTime = Instant.now();
+    }
+
+    public String getId() {
```

Comment:
off topic: should the job id of our use case be something like "Timestamp-SiteId(-DataType-GlobalMetadataVersionGeneratedFrom)"?

Reply:
I think this can be implemented as an abstract class that implements the interface for sync jobs

i.e. `public abstract class SyncJob implements Job` and the job ID you mentioned can be part of the constructor?

Reply:
yeh i think we should identify the job easily by forming the job id with info above cc @aulme